test/if-then-else.test.js

```diff
@@ -249,15 +249,7 @@ const alphabetOutput = JSON.stringify({
 const deepFoobarOutput = JSON.stringify({
   foobar: JSON.parse(foobarOutput)
 })
-const noElseGreetingOutput = JSON.stringify({
-  kind: 'greeting',
-  foo: 'FOO',
-  bar: 42,
-  hi: 'HI',
-  hello: 45,
-  a: 'A',
-  b: 35
-})
+const noElseGreetingOutput = JSON.stringify({})
```

Comment:
Why did you change this?

Reply:
Because [the test](https://github.com/fastify/fast-json-stringify/blob/69a34a2d3d4670b8641e1ecd9df9458f30819986/test/if-then-else.test.js#L311) was wrong. 

Here is a schema: https://github.com/fastify/fast-json-stringify/blob/69a34a2d3d4670b8641e1ecd9df9458f30819986/test/if-then-else.test.js#L167

Here is an input data: https://github.com/fastify/fast-json-stringify/blob/69a34a2d3d4670b8641e1ecd9df9458f30819986/test/if-then-else.test.js#L209

As you can see the `if` condition doesn't fullfil as `greetingInput.kind` equal `greeting` and not a `foobar`, so it should use `else` schema instead of `then`. And as `else` schema is missing the result is an empty object.